src/lbPair.ts

```diff
@@ -86,30 +83,48 @@ export function handleSwap(event: SwapEvent): void {
   const tokenXPriceUSD = tokenX.derivedAVAX.times(bundle.avaxPriceUSD);
   const tokenYPriceUSD = tokenY.derivedAVAX.times(bundle.avaxPriceUSD);
 
-  const swapForY = event.params.swapForY;
-  const tokenIn = swapForY ? tokenX : tokenY;
-  const tokenOut = swapForY ? tokenY : tokenX;
+  // const swapForY = event.params.swapForY;
+  const tokenIn = tokenX;
```

Comment:
I believe tokenIn can be either tokenX or tokenY, depending on the swap direction in barn we something like:
```
swapForY = amountXIn != 0 and amountYIn == 0 
```

Reply:
seems like this may not matter based on how the code is written